lib/ui/home/tabs/store_tab.dart

```diff
@@ -1,22 +1,253 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'package:cobble/domain/connection/connection_state_provider.dart';
+import 'package:cobble/domain/entities/hardware_platform.dart';
+import 'package:cobble/ui/common/components/cobble_button.dart';
+import 'package:cobble/ui/common/icons/fonts/rebble_icons.dart';
+import 'package:cobble/ui/common/icons/watch_icon.dart';
 import 'package:cobble/ui/router/cobble_scaffold.dart';
 import 'package:cobble/ui/router/cobble_screen.dart';
+import 'package:cobble/ui/theme/with_cobble_theme.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:package_info/package_info.dart';
+import 'package:url_launcher/url_launcher.dart';
 import 'package:webview_flutter/webview_flutter.dart';
+import 'package:cobble/infrastructure/pigeons/pigeons.g.dart';
+import 'package:cobble/domain/apps/app_manager.dart';
 
-class StoreTab extends StatefulWidget implements CobbleScreen {
-  @override
-  State<StatefulWidget> createState() => new _StoreTabState();
-}
+class StoreTab extends HookWidget implements CobbleScreen {
+  final String baseUrl = "https://store-beta.rebble.io";
+  final Completer<WebViewController> _controller =
```

Comment:
`_controller` and `searchController` should be created inside build method. Also, check out [useTextEditingController](https://pub.dev/documentation/flutter_hooks/latest/flutter_hooks/useTextEditingController-constant.html).

Reply:
Oh damn, that's super useful